src/hooks/useManifestState/reducer.js

```diff
@@ -102,7 +102,7 @@ const updateState = (state, action) => {
   const updatedState = { ...state }
 
   if (action.filter) {
-    if (action.filter !== updatedState.filter) {
+    if (JSON.stringify(action.filter) !== JSON.stringify(updatedState.filter)) {
```

Comment:
I would like to keep this an identity check if possible.

Reply:
It does not seem to be working as an identity check can the 2 objects ever have the same identity?